lib/models/project.js

```diff
@@ -73,20 +74,14 @@ function Project(root, pkg, ui, cli) {
   @method setupBowerDirectory
  */
 Project.prototype.setupBowerDirectory = function() {
-  var bowerrcPath = path.join(this.root, '.bowerrc');
-
-  logger.info('bowerrc path: %s', bowerrcPath);
-
-  if (existsSync(bowerrcPath)) {
-    try {
-      this.bowerDirectory = fs.readJsonSync(bowerrcPath).directory;
-    } catch (exception) {
-      logger.info('failed to parse bowerc: %s', exception);
-      this.bowerDirectory = null;
-    }
+  var config;
+  try {
+    config = bowerConfig.read(this.root);
+  } catch (e) {
+    config = {directory: 'bower_components'};
```

Comment:
I was merely following the existing code, that falls back to `bower_components` if the `.bowerrc` is malformed?

How could we handle this otherwise?